doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -240,63 +275,11 @@ Error list (HTTP response code in paranthesis):
 * LengthRequired (411). Zero/No Content-Length in PUT/POST/PATCH request
 * RequestEntityTooLarge (413). Payload too large
 * UnsupportedMediaType (415). Request content type is not supported.
-* InvalidModification (422). Some piece of information is missing in payload.
```

Comment:
Probably it doesn't matter as the Error List will be rewritten but... why the 422 has been removed in this PR?